src/MgvHasOffers.sol

```diff
@@ -74,11 +74,6 @@ contract MgvHasOffers is MgvCommon {
         // with
         // offerId == local.best() (but best will maybe go away in the future)
 
-        // If shouldUpdateBranch is false is means we are about to insert anyway, so no need to load the best branch right now
-        // if local.bin < offerBin then a better branch is already cached. note that local.bin >= offerBin implies local.bin = offerTick
-        // no need to check for prevId/nextId == 0: if offer is last of leaf, it will be checked by leaf.isEmpty()
```

Comment:
Should the comment be retained? I see it's a bit deprecated, but still nice to have the explanation.

Reply:
It's being replaced by a comment in another branch :)